hack/test-go.sh

```diff
@@ -92,6 +94,12 @@ shift $((OPTIND - 1))
 # Use eval to preserve embedded quoted strings.
 eval "goflags=(${KUBE_GOFLAGS:-})"
 
+if [[ -n "${1-}" ]]; then
+  test_dirs=$@
```

Comment:
This assignment is never correct. This should be an array. (Yes, I know it's pre-existing.) But there's a better way to handle it.